frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -1541,6 +1564,16 @@ private void testPredictions(String modelName, String expectedOutput, String ver
 
         TestUtils.getLatch().await();
         Assert.assertEquals(TestUtils.getResult(), expectedOutput);
+
+        Channel metricsChannel = TestUtils.getMetricsChannel(configManager);
+        TestUtils.setResult(null);
+        TestUtils.setLatch(new CountDownLatch(1));
+        DefaultFullHttpRequest metricsReq =
+                new DefaultFullHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.GET, "/metrics");
+        metricsChannel.writeAndFlush(metricsReq);
+        TestUtils.getLatch().await();
+        Pattern inferLatencyMatcher = TestUtils.getTSInferLatencyMatcher(modelName, version);
+        Assert.assertTrue(inferLatencyMatcher.matcher(TestUtils.getResult()).find());
```

Comment:
Add UTs for all types of parameters supported by the metrics API.

Reply:
Done